static/js/admin.js

```diff
@@ -118,6 +118,14 @@ function populate_streams(streams_data) {
     loading.destroy_indicator($('#admin_page_streams_loading_indicator'));
 }
 
+function toggle_email_change_display() {
+    $("#settings_overlay_container li[data-section='your-account']").click();
+    $("#change_email").toggle();
+    $(".icon-vector-question-sign").toggle();
+    $("#settings_overlay_container").addClass("show");
+    $("#settings_overlay_container li[data-section='organization-settings']").click();
+}
```

Comment:
I am sorry for making such mistake. I think i have seen a similar piece of code somewhere doing a similar thing. I guess that code is also irrelevant then. Ping you one i find it. :)